source/features/bugs-tab.tsx

```diff
@@ -11,6 +12,20 @@ import {getRepo} from '../github-helpers';
 import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 
+async function highlightBugsTabOnIssuePage(): Promise<void | false> {
+	if (await countBugs() > 0 && !await elementReady('.sidebar-labels .IssueLabel[href$="/bug" i]')) {
+		return false;
+	}
+
+	await delay(1); // Wait until the bug tab is created
```

Comment:
This is arbitrary. We should use `select(tab) ?? await oneMutation(tabBar, filter)`

Reply:
What about
```suggestion
	await elementReady('.rgh-bug-tab', {waitForDomReady: false})
```

This is really the same thing as oneMutation

Reply:
hmmm this might be just about the only case where `waitForDomReady: false` is acceptable, since we handle everything in this file. I mean, not ideal, still, but maybe ok enough.